src/shared/directives/conditional-show/domain/conditional-show.model.spec.ts

```diff
@@ -514,7 +515,7 @@ describe('conditional-show', () => {
 
     it('should return true when all conditions are true', () => {
       caseField1.value = ['s1', 's2', 's3'];
-      let sc = new ShowCondition('field1CONTAINS"s3,s2" AND field2=3 AND field3="te*" AND field4="s1 AND s2"');
+      let sc = new ShowCondition('field1 CONTAINS"s3,s2" AND field2=3 AND field3="te*" AND field4="s1 AND s2"');
```

Comment:
Same as the comment above about operators not needing to have spaces before or after...

Reply:
As I mentioned in my earlier comment